appinventor/appengine/src/com/google/appinventor/client/OdeMessages.java

```diff
@@ -349,7 +330,7 @@
   // Used in Ode.java
 
   // TODO(user): Replace with commented version once we're ready
-  @DefaultMessage("MIT App Inventor 2")
+  @DefaultMessage("MIT App Inventor")
```

Comment:
I don't have a problem with it, but we should probably separate it out into its own commit.